app/models/car.rb

```diff
@@ -0,0 +1,7 @@
+class Car < ApplicationRecord
+  has_many :reservations, dependent: :destroy
+  validates :name, presence: true
+  validates :description, presence: true
+  validates :photo, presence: true
```

Comment:
[OPTIONAL] Consider putting some validation since it'd be a url

Reply:
please complete the review quickly as we have no time to spare.